### ISSUE TITLE
Request: Add an install option to RoT to use Drizzt Saga portraits

### ISSUE THREAD

**Issue by Endarire** [SpellholdStudios/Region_of_Terror#48]
Greetings, all, especially @weigo87 and @Roberciiik!

For the sake of consistency, may we get an install option to use the [Drizzt Saga](https://github.com/Black-Wyrm-Lair/Drizzt-Saga/) portraits for the characters in each of these mods?  It can be done via manual replacement, but including it as an official install option is better.

Thankee!

**Comment by Roberciiik** [comment]
I think this feature could go into "One Drizzt" modification that is installed after these two mods.

**Comment by ALIENQuake** [comment]
I agree with Roberciiik, @Endarire when SHS is back you can post this suggestion in the One Drizzt" modification thread or poke me to do it.
